kill-ryzen.sh

```diff
@@ -23,34 +24,42 @@ if which apt-get &>/dev/null; then
 elif which dnf &>/dev/null; then
   sudo dnf install -y @development-tools
 elif which pacman &>/dev/null; then
+  echo "You are using a rolling release distribution based on ArchLinux. You are on your own if things fail."
   sudo pacman -S --needed base-devel
 else
-  exit 1
+  echo "Your distribution is not officially supported. The script will continue but may fail if you don't have the required build tools. You are on your own if things fail."
 fi
 
 if $USE_RAMDISK; then
-  echo "Create compressed ramdisk"
-  sudo mkdir -p /mnt/ramdisk || exit 1
-  sudo modprobe zram num_devices=1 || exit 1
-  echo 64G | sudo tee /sys/block/zram0/disksize || exit 1
-  sudo mkfs.ext4 -q -m 0 -b 4096 -O sparse_super -L zram /dev/zram0 || exit 1
-  sudo mount -o relatime,nosuid,discard /dev/zram0 /mnt/ramdisk/ || exit 1
-  sudo mkdir -p /mnt/ramdisk/workdir || exit 1
-  sudo chmod 777 /mnt/ramdisk/workdir || exit 1
-  cp buildloop.sh /mnt/ramdisk/workdir/buildloop.sh || exit 1
-  cd /mnt/ramdisk/workdir || exit 1
-  mkdir tmpdir || exit 1
+  #Create compressed ramdisk if it doesn't already exist
+  if mountpoint -q $MOUNTPOINT; then
+    echo "RAMdisk already mounted on $MOUNTPOINT."
+  else
+    echo "Create compressed ramdisk"
+    sudo mkdir -p $MOUNTPOINT || exit 1
+    sudo modprobe zram num_devices=1 || exit 1
+    # Setting maximum RAM disk size
+    echo 64G | sudo tee /sys/block/zram0/disksize || exit 1
+    sudo mkfs.ext4 -q -m 0 -b 4096 -O sparse_super -L zram /dev/zram0 || exit 1
+    sudo mount -o relatime,nosuid,discard /dev/zram0 $MOUNTPOINT || exit 1
+  fi
+  sudo mkdir -p $MOUNTPOINT/workdir || exit 1
+  sudo chmod 777 $MOUNTPOINT/workdir || exit 1
+  cp buildloop.sh $MOUNTPOINT/workdir/buildloop.sh || exit 1
+  cd $MOUNTPOINT/workdir || exit 1
+  mkdir -p tmpdir || exit 1
   export TMPDIR="$PWD/tmpdir"
 fi
 
 echo "Download GCC sources"
-wget ftp://ftp.fu-berlin.de/unix/languages/gcc/releases/gcc-7.1.0/gcc-7.1.0.tar.bz2 || exit 1
+DOWNLOAD_FILE=gcc-7.2.0.tar.xz
+wget -O="$DOWNLOAD_FILE" -c ftp://ftp.fu-berlin.de/unix/languages/gcc/releases/gcc-7.2.0/$DOWNLOAD_FILE || exit 1
```

Comment:
I think there is a typo here.  It's either:
$ wget -O "$DOWNLOAD_FILE"
or
$ wget --output-document="$DOWNLOAD_FILE"

Changing it fixed the script for me.

Reply:
thank you for your feedback. Strangely, it is working as it on my side, but I think that you are right about the superfluous "=" between "-O" and "$DOWNLOAD_FILE".

Reply:
Fixed.